include/pybind11/embed.h

```diff
@@ -168,6 +171,9 @@ inline void initialize_interpreter(bool init_signal_handlers = true,
         throw std::runtime_error(PyStatus_IsError(status) ? status.err_msg
                                                           : "Failed to init CPython");
     }
+#    if defined(WITH_THREAD) && PY_VERSION_HEX < 0x03070000
```

Comment:
This is in the `else` portion of `#if PY_VERSION_HEX < 0x030B0000`.
`PY_VERSION_HEX >= 0x030B0000 && PY_VERSION_HEX < 0x03070000` seems like it'd always be false.

If so, consider removing?

Reply:
Yes, thanks for catching that! Sorry I was going through with lawnmower mentality and completely missed that.

Reply:
Welcome!